packages/frontend/web-ui/src/auth/hooks/useRegister.ts

```diff
@@ -192,6 +200,7 @@ export const useRegister = (): [UseRegisterData, UseRegisterActions] => {
     } else {
       setRegisterData({
         form: {
+          errorMessage: '',
```

Comment:
```suggestion
          errorMessage: undefined,
```
An empty string error is still an error. We have no error from the endpoint at this point, right?

Reply:
You're right! I've deleted this line.